app/decorators/event_decorator.rb

```diff
@@ -0,0 +1,19 @@
+class EventDecorator < Draper::Decorator
+  delegate :picture, :title, :place, :beginning_at, :end_at, :description, :category
+
+  def self.collection_decorator_class
+    PaginatingDecorator
+  end
+
+  def beginning_at
+    I18n.l object.beginning_at, format: :datetime
+  end
+
+  def end_at
+    I18n.l object.beginning_at, format: :datetime
+  end
+
+  def short_description
+    h.truncate(object.description, length: 400, separator: ' ')
```

Comment:
`description` is delegated to object so you can omit `object.` and write just `description`